backend/src/routes/auth.ts

```diff
@@ -60,17 +67,19 @@ export const authRoute = new Elysia({ prefix: '/auth' })
                 };
             }
 
-            const { email, password } = validEmaillPass.data;
-
+            const { email, password, name, surname, phone } = validEmaillPass.data;
```

Comment:
Can you format phone number, maybe just replace all "-" with ""
so 123-456-7890 cannot be use again with 1234567890